### ISSUE TITLE
Not seeing sensors after install

### ISSUE THREAD

**Issue by Beenay25** [BottlecapDave/HomeAssistant-OctopusEnergy#29]
Hi,

I'm probably missing something obvious.

I've copied the "octopus_energy" folder that contains all the .py files to custom_components

I've confirmed that I can see said folder/files in the Home Assistant file editor.

I then followed the UI link in the instructions and added the Octopus Energy integration.

I gave it my API key and account reference. I didn't select "is SMETS1" as both my gas and electricity meters are SMETS2

I've restarted Home Assistant.

I do not see any of the mentioned sensors under DeveloperTools/States (or anywhere else)
I've tried searching the entire list for "octopus" (so that sensors like sensor.octopus_energy_electricity_current_rate would match)

Can anyone suggest what I've missed?

**Comment by BottlecapDave** [comment]
Sorry to hear you're having issues. Are you seeing any errors in your Home Assistant logs?

**Comment by Beenay25** [comment]
Does the following help?

`2021-12-15 14:26:47 WARNING (SyncWorker_0) [homeassistant.loader] We found a custom integration octopus_energy which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-12-15 14:26:47 WARNING (SyncWorker_1) [homeassistant.loader] We found a custom integration httas which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-12-15 14:26:47 ERROR (SyncWorker_1) [homeassistant.loader] The custom integration 'httas' does not have a version key in the manifest file and was blocked from loading. See https://developers.home-assistant.io/blog/2021/01/29/custom-integration-changes#versions for more details
2021-12-15 14:26:47 WARNING (SyncWorker_2) [homeassistant.loader] We found a custom integration nodered which has not been tested by Home Assistant. This component might cause stability problems, be sure to disable it if you experience issues with Home Assistant
2021-12-15 14:26:50 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [2966005912] Received invalid command: energy/info
2021-12-15 14:26:50 ERROR (MainThread) [homeassistant.components.websocket_api.http.connection] [2966005912] Received invalid command: energy/info
2021-12-15 14:26:59 ERROR (MainThread) [homeassistant.components.sensor] Error while setting up octopus_energy platform for sensor
Traceback (most recent call last):
File "/usr/src/homeassistant/homeassistant/helpers/entity_platform.py", line 249, in _async_setup_platform
await asyncio.shield(task)
File "/config/custom_components/octopus_energy/sensor.py", line 40, in async_setup_entry
await async_setup_default_sensors(hass, entry, async_add_entities)
File "/config/custom_components/octopus_energy/sensor.py", line 67, in async_setup_default_sensors
entities.append(OctopusEnergyLatestElectricityReading(client, point["mpan"], meter["serial_number"]))
NameError: name 'OctopusEnergyLatestElectricityReading' is not defined`
